webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/NetworkingInterfaces/NICDescriptionList.js

```diff
@@ -0,0 +1,132 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import {
+  Label,
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  Button,
+  Grid,
+  GridItem,
+  Accordion,
+  AccordionItem,
+  AccordionToggle,
+  AccordionContent,
+} from '@patternfly/react-core';
+import { TagIcon, HddIcon } from '@patternfly/react-icons';
+import { translate as __ } from '../../../../../../common/I18n';
+import { STATUS } from '../../../../../../constants';
+
+const types = {
+  interface: __('Interface'),
+  bridge: __('Bridge'),
+  bmc: __('BMC'),
+};
+const NICDescriptionList = ({ status, foremanInterface }) => {
+  const {
+    name,
+    primary,
+    provision,
+    type,
+    mac,
+    mtu,
+    ip,
+    ip6,
+    fqdn,
+    subnet_id: subnetID,
+    subnet_name: subnetName,
+    subnet6_id: subnet6ID,
+    subnet6_name: subnet6Name,
+  } = foremanInterface;
+  const [isExpanded, setIsExpanded] = useState(primary);
+  return (
+    <Accordion className="interface-accordion">
+      <AccordionItem>
+        <AccordionToggle
+          onClick={() => {
+            setIsExpanded(curr => !curr);
+          }}
+          isExpanded={isExpanded}
+        >
+          <Grid>
+            <GridItem span={7} className="interface-name">
+              {name || <span className="disabled-text">{__('No name')}</span>}
+              {primary && <TagIcon title={__('Primary')} />}
+              {provision && <HddIcon title={__('Provision')} />}
+            </GridItem>
+            <GridItem span={4}>
+              <Label color="blue">{types[type] || type}</Label>
+            </GridItem>
+          </Grid>
```

Comment:
Maybe this would work better as a `<Flex>`? That way items could wrap. I'm getting some crowding:
![image](https://user-images.githubusercontent.com/22042343/198387043-2c881aa1-c44c-4285-8698-b4a817a8f599.png)

Reply:
Can you please check if its still an issue? switched to a compact label and in my env the text gets wrapped before it gets this crowded

Reply:
It still collides if the identifier is really long. But it's not as much of a problem now since identifiers are short.